core/ecschema-rpc/impl/README.md

```diff
@@ -0,0 +1,9 @@
+﻿# iModel.js RPC Interface Integration Tests
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This package contains the Schema RPC Interface Implementation which can be added to a backend.
+
+## Testing
+
+The tests for this RPC Interface are available in the @bentley/schema-rpcinterface-tests package, please visit the README in that package for more information.
```

Comment:
Where is this package located? Are those tests different from the ones in this package?

Reply:
This package is located within the ecschema-rpc folder, in particular the integration-tests folder. I just added a hyperlink into the readme so anyone can navigate directly to it.